src/main/resources/assets/computercraft/lang/it_it.lang

```diff
@@ -0,0 +1,83 @@
+upgrade.minecraft:diamond_shovel.adjective=Scavatrice
+upgrade.minecraft:diamond_pickaxe.adjective=Minatrice
+upgrade.minecraft:diamond_axe.adjective=Taglialegna
+upgrade.minecraft:diamond_hoe.adjective=Contadina
+upgrade.computercraft:wireless_modem.adjective=Wireless
+upgrade.minecraft:crafting_table.adjective=Artigiano
+upgrade.computercraft:advanced_modem.adjective=Ender
+upgrade.computercraft:speaker.adjective=Rumoroso
+
+gui.computercraft:wired_modem.peripheral_connected=Periferica "%s" connessa alla rete
+gui.computercraft:wired_modem.peripheral_disconnected=Periferica "%s" disconessa dalla rete
+
+gui.computercraft:tooltip.copy=Copia negli appunti
+
+gui.computercraft:config.computer_space_limit=Limite spazio Computer (bytes)
+gui.computercraft:config.floppy_space_limit=Limite spazio Disco Floppy (bytes)
+gui.computercraft:config.maximum_open_files=Massimo file aperti per computer
+gui.computercraft:config.disable_lua51_features=Disattiva features Lua 5.1
+gui.computercraft:config.default_computer_settings=Impostazioni Computer predefinite
+gui.computercraft:config.debug_enabled=Attiva libreria di debug
+gui.computercraft:config.computer_threads=Threads computer
+gui.computercraft:config.log_computer_errors=Salva errori computer
+
+gui.computercraft:config.http=HTTP
+gui.computercraft:config.http.enabled=Attiva l'API HTTP
+gui.computercraft:config.http.websocket_enabled=Attiva websocket
+gui.computercraft:config.http.whitelist=Lista bianca HTTP
+gui.computercraft:config.http.blacklist=Lista nera HTTP
+
+gui.computercraft:config.http.timeout=Tempo di scadenza
+gui.computercraft:config.http.max_requests=Richieste correnti massime
+gui.computercraft:config.http.max_download=Massimo grandezza risposte
+gui.computercraft:config.http.max_upload=Massimo 
+gui.computercraft:config.http.max_websockets=Connessioni websocket massime
+gui.computercraft:config.http.max_websocket_message=Massimo grandezza messaggi websocket
+
+gui.computercraft:config.peripheral=Periferiche
+gui.computercraft:config.peripheral.command_block_enabled=Attiva periferica blocco comandi
+gui.computercraft:config.peripheral.modem_range=Raggio Modem (default)
+gui.computercraft:config.peripheral.modem_high_altitude_range=Raggio Modem (Altitudine alta)
+gui.computercraft:config.peripheral.modem_range_during_storm=Raggio Modem (brutto tempo)
+gui.computercraft:config.peripheral.modem_high_altitude_range_during_storm=Raggio Modem (Altitudine alta, brutto tempo)
+gui.computercraft:config.peripheral.max_notes_per_tick=Note massime un computer può suonare alla volta
+
+gui.computercraft:config.turtle=Tartarughe
+gui.computercraft:config.turtle.need_fuel=Attiva carburante
+gui.computercraft:config.turtle.normal_fuel_limit=Limite carburante tartarughe
+gui.computercraft:config.turtle.advanced_fuel_limit=Limite carburante tartarughe avanzate
+gui.computercraft:config.turtle.obey_block_protection=Le tartarughe obbediscono alle protezioni dei blocchi
```

Comment:
As above, this causes the config GUI to overflow:

![image](https://user-images.githubusercontent.com/4346137/54294957-0c9ec780-45aa-11e9-85a1-3cb6fa472e50.png)

It's definitely not the end of the world, as few people actually use it, but may be worth considering shorter forms.